### ISSUE TITLE
Windows compatibility

### ISSUE THREAD

**Issue by ramirofages** [qnzhou/PyMesh#18]
Hi! First of all, awesome library. I wanted to ask if it is working correctly on windows. I downloaded it with pip install but when I use the method load_mesh, an error pops up telling me that it is not defined. When I checked the files in the instalation folder I feel like there are missing files. Do you know something about this? 

Cheers

**Comment by qnzhou** [comment]
Thanks for the support!  Windows support has been on my agenda for a while, but I have not get a chance to work on it yet.  Sorry!

On a separate note, the [pymesh package](https://pypi.python.org/pypi/pymesh/1.0.2) you installed from pip is not this one.  It is https://github.com/taxpon/pymesh.  Unfortunately, we use the same package name, and Takuro is one step ahead of me in registering the package on pypi.

**Comment by ramirofages** [comment]
Oh lol I see, then I have a question, what does it need to work on windows? Currently for my job I need to calculate the gaussian curvature of a mesh, and if it isn't too much I may be able to make it work on windows. Cheers!

**Comment by ramirofages** [comment]
Hey! Sorry to bother you, but I'm not very proficient with compiling libraries, is there any way that you could provide me the compiled code to run it on linux (any distro)?

**Comment by qnzhou** [comment]
Sorry about the lack of responses.  I am looking into it.  Having a binary should be possible.

**Comment by kuttamuwa** [comment]
Hey, 
I want to ask when will binary be released?

**Comment by flutefreak7** [comment]
I'll chime in to express interest in this as well!

Pymesh looks amazing.

**Comment by brainmachine** [comment]
Hey, I'm using PyMesh in a docker container (which is great for prototyping!). Now I need to compile it on Windows 10 but I haven't had much luck.    
Does anyone know what is needed to make this work?

Thanks for making PyMesh! It's a really cool library.

**Comment by souvikmeister** [comment]
Hi Guys,
Seems like Pymesh is still not available on windows10. Any chance this can happen soon ? It is a really cool library and will benefit many people if moved to a win64 environment soon or instructions provided on what to do to make it happen.
Thanks.
Souvik

**Comment by sama2689** [comment]
I agree with the comment on windows 10. I found a [github repository](https://github.com/rickarkin/PyMesh-win) of what looks to be a windows compatible version of the package but the instructions are not updated to be compatible with windows?
